projects/project3/src/main/scala/net/sansa_stack/template/spark/rdf/TripleReader.scala

```diff
@@ -0,0 +1,58 @@
+package net.sansa_stack.template.spark.rdf
+
+import java.net.URI
+
+import scala.collection.mutable
+
+import net.sansa_stack.rdf.spark.io._
+import org.apache.spark.sql.SparkSession
+import org.apache.jena.riot.Lang
+
+object TripleReader {
```

Comment:
I think you could clean a bit your project. Consider removing this class, as it hasn't been used elsewhere.